frontend/src/i18n/en/translation_old.json

```diff
@@ -0,0 +1,48 @@
+{
+    "tooltip": {
+        "explore-menu": "Get a random selection of reading experiences and annotations",
+        "open-related": "View items that are related to the current item",
+        "open-annotations": "View all places where this item has been annotated",
+        "open-external": "View links to external resources about this item",
+        "edit-annotation": "Change the properties of this item",
+        "clone-annotation": "Create a new annotation at the same position"
+    },
+    "annotation": {
+        "edit-info": {
+            "begin": "(Find more info on what and how to annotate in "
+        }
+    },
+    "feedback": {
+        "failed": "Feedback submission failed. Please try again and if this error keeps occuring, contact the site administrator."
+    },
+    "property": {
+        "typeHints": {
+            "integer": "Include extra leading zeros to force interpretation as an integer type.",
+            "base64Binary": "Include internal whitespace to force interpretation as base 64 binary data.",
+            "gYear": "Include time zone information (such as a final &ldquo;Z&rdquo;) to force interpretation as a Gregorian year.",
+            "string": "Start with a space to force interpretation as text or a derived string type."
+        },
+        "xsd-available-types": {
+            "begin": "See ",
+            "end": " for an overview of the available types and their lexical representations."
+        }
+    },
+    "interpunction": {
+        "colon": ": "
+    },
+    "not-found": {
+        "text": {
+            "begin": "Oops, this page doesn't seem to exist. You can ",
+            "between-back-landing": ", visit ",
+            "between-landing-search": " or "
+        }
+    },
+    "source": {
+        "delete-confirm": "\nAre you sure you want to delete this source?\nIf you delete this source, all its annotation will be deleted as well, including any annotations that other users may have made.\nThis cannot be undone.\n"
+    },
+    "register": {
+        "thanks": {
+            "begin": "Thank you for signing up. You can now "
+        }
+    }
+}
```

Comment:
Do we need this file?

Reply:
Yes, it prevents the contained strings from becoming empty in the non-old version.